syntax = "proto3";

package statemanager;

import "constants.proto";

service Connection {
  rpc Send (SendRequest) returns (SendResponse);
}

message SendRequest {
  constants.PiccoloModuleName from = 1;
  string request = 2;
}

message SendResponse {
  string response = 1;
}
